deploy/rbd/helm/templates/provisioner-statefulset.yaml

```diff
@@ -46,8 +48,8 @@ spec:
           imagePullPolicy: {{ .Values.nodeplugin.plugin.image.pullPolicy }}
           args:
             - "--csi-address=$(ADDRESS)"
-            - "--connection-timeout=15s"
             - "--v=5"
+            - "--timeout=60s"
```

Comment:
let's keep the timeout constant in all sidecars.

Reply:
The reason is pretty simple, the snapshotter side car default is already 60s, but the option line states this as 10s (see [this](https://github.com/kubernetes-csi/external-snapshotter/blob/ac33959738598e55a0a0a9d5a2d08d7445f27f36/cmd/csi-snapshotter/main.go#L69)). I wanted to retain it at 60s for now even if the snapshotter side car corrects the same. Also, the snapshot is attempted only once, and not retried on timeouts, hence higher timeouts for the same.

Reply:
the comment looks wrong. the default value is 1 minute, yes I agree, I would  suggest keeping all  sidecar container  timeout  to  60sec

Reply:
comment already fixed in master https://github.com/kubernetes-csi/external-snapshotter/blob/master/cmd/csi-snapshotter/main.go#L69

Reply:
What is required finally? 60s everywhere? That should be fine as well.

Reply:
yes please make it 60s in other places too

Reply:
Updated in the latest push.